### ISSUE TITLE
Exception: "Do not add the "sitemap" loader twice in ."

### ISSUE THREAD

**Issue by honigkuchenbaer** [BitBagCommerce/SyliusCmsPlugin#291]
Hi, from time to time this exception pops up when I change something in my custom routing.yml. Apparently the whole routing config will be re-assembled then and fails due to some imported sitemap-routing.yml.

Has anyone a solution for this?

`An exception has been thrown during the rendering of a template ("Do not add the "sitemap" loader twice in . (which is being imported from "/var/www/vendor/stefandoorn/sitemap-plugin/src/Resources/config/routing.yml"). Make sure there is a loader supporting the "sitemap" type.").`

This is how I do import routing.yml from BitBagCmsPlugin:

```
bitbag_sylius_cms_plugin:
    resource: "@BitBagSyliusCmsPlugin/Resources/config/routing.yml"
```

**Comment by stefandoorn** [comment]
To me this happens when already loading the sitemap plugin yourselves. Then it will be loaded another time via this plugin. I'm not sure though what is the best solution to prevent this.

Any clue @bitbager? As soon as I remove one of the calls to loading the routing file for the sitemap plugin, it works fine again.

**Comment by stefandoorn** [comment]
This might actually be the solution: https://github.com/stefandoorn/sitemap-plugin/pull/123

**Comment by dark774** [comment]
Hello

**Comment by leszczuu** [comment]
https://github.com/BitBagCommerce/SyliusCmsPlugin/pull/327
removed including sitemap config from cms plugin.

**Comment by leszczuu** [comment]
fixed with 3.0.0 release
